server/routerlicious/packages/protocol-definitions/src/storage.ts

```diff
@@ -44,6 +44,10 @@ export interface IBlob {
     encoding: string;
 }
 
+export interface IAttachment {
```

Comment:
I'd use more generic name to represent any kind of ID (blob/tree/etc.).
That said, how do we represent reused blobs in summaries today on r11s? Can we simply use that mechanism here?
+ @arintwt

Reply:
Although we don't do it currently, summary handles should be able to point to existing blobs.